src/components/Notifications/BehaviorGroup/BehaviorGroupSaveModal.tsx

```diff
@@ -19,7 +19,7 @@ interface InternalProps {
 }
 
 const InternalBehaviorGroupSaveModal: React.FunctionComponent<InternalProps> = props => {
-    const title =  `${props.data ? 'Create new' : 'Edit'} behavior group`;
+    const title =  `${props.data?.actions ? 'Edit' : 'Create new'} behavior group`;
```

Comment:
Could you check for the `id` instead of the actions?

It should be more stable - I'm wondering what would happen if you
1- Click "Create new group"
2- Add an action

Reply:
hm, nothing seemed to change here. I didn't see/get any error when adding actions, but changed to check id!

Reply:
Ah ok - anyhow the `id` should be more future proof. thanks!